src/ansys_sphinx_theme/templates/changelog_template.jinja

```diff
@@ -0,0 +1,36 @@
+{% if sections["Whatsnew"] %}
+  {%+ for category, val in sections["Whatsnew"].items() +%}
+    {%+ if definitions.get(category) and definitions[category].get('name') +%}
+{{ definitions[category]['name'] }}
+{{ "-" * definitions[category]['name']|length }}
+    {% endif %}
+    {%- for text, values in sections["Whatsnew"][category].items() %}
```

Comment:
Are you using the `values` variable in here?

Reply:
I can add it if we want to add/remove or we can also give it as an option to the user

Reply:
Is it just the issue number or the link to the issue? Can we create a link with it?

Reply:
yes, it is issue number with link. we can add that values also in the template